docs/DeveloperGuide.adoc

```diff
@@ -295,25 +297,294 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |user |make sure that my notes doesn’t miss out important points |collate and compare my notes with my friends’ efficiently
+
+|`* *` |user |be reminded at appropriate times to revise my work |revise regularly and consistently
+
+|`* *` |user |disseminate information efficiently |save the work amongst different teammates
+
+|`* *` |user |better understand or memorize the topics |improve my results
+
+|`* *` |user |connect different parts of questions together | easily link concepts together
+
+|`* *` |user |share notes with my friends and for them to share notes with me, for ease of discussion |be able to study with others
+
+|`* *` |user |answer the questions within a certain time frame |boost my confidence and proficiency in a subject
+
+|`* *` |user |keep track of the harder questions |revise more effectively
+
+|`* *` |university educator |keep track of my students' progress and evaluate who are the stronger/weaker students so they can help each other |help my students do well in their studies
+
+|`* *` |university educator |distribute the flashcards I create to my students so they can use them to practice |help my students better memorise and recall my class content
+
+|`* *` |university educator |give my students quizzes during lecture using the flashcards | make sure that they will not fall asleep during lecture.
+
+|`*` |user |download flashcards from online sources |save time for other revisions
+
+|`*` |user |create notes without papers | save the earth
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `StudyBuddy` and the *Actor* is the `user`, unless specified otherwise)
+
+[discrete]
+=== *User story 1:*
+As a student who is busy with revision and has no time to create a cheatsheet, the cheatsheet
+generation feature will help me save time on creating cheatsheets and let me spend more time on revision.
+
+[discrete]
+=== Use case: UC01 – Create a cheatsheet
+
+Preconditions: StuddyBuddy application is opened
+
+*MSS*
+
+1.	User navigates to cheetsheat section.
+
+2.	User chooses to create new cheatsheet.
+
+3.	StuddyBuddy asks user for new cheatsheet’s title.
+
+4.	User inputs new cheatsheet’s title.
+
+5.	StuddyBuddy asks user for new cheatsheet’s module.
+
+6.	User inputs new cheatsheet’s module.
+
+7.	StuddyBuddy asks for new cheatsheet’s tags.
+
+8.	User inputs new cheatsheet’s tags.
++
+Use case ends
+
+[discrete]
+=== Use case: UC02 – List cheatsheets
+
+Preconditions: StuddyBuddy application is opened
+
+*MSS*
+
+1.	User navigates to cheetsheat section.
+
+2.	User chooses to list all cheatsheet.
+
+3.	StudyBuddy displays all cheatsheets.
```

Comment:
Possible Extension:
3a StudyBuddy display empty panel, with message "No cheatsheet to display"
Use case end